src/content/configuration/dev-server.md

```diff
@@ -1108,6 +1109,22 @@ Usage via the CLI
 webpack-dev-server --quiet
 ```
 
+## `devServer.serveIndex`
+
+`boolean`
+
+This option Enables/Disables [`serveIndex`](https://github.com/expressjs/serve-index) middleware. It's enabled by default.
```

Comment:
```suggestion
Tells dev-server to use [`serveIndex`](https://github.com/expressjs/serve-index) middleware when enabled.
```

Reply:
@EugeneHlushko Done 😄

Reply:
thank you, By the way, you can directly commit proposed changes via github interface button: "Commit suggestion"